lambdas/sonar-project.properties

```diff
@@ -0,0 +1,18 @@
+sonar.projectKey=alphagov_di-ipv-cri-common-lambdas_typescript
+sonar.organization=alphagov
+sonar.projectBaseDir=lambdas
+
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
+
+
+
+
+# This is the name and version displayed in the SonarCloud UI.
+#sonar.projectName=alphagov_di-ipv-cri-common-lambdas_typescript
+#sonar.projectVersion=1.0
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+#sonar.sources=.
+
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
```

Comment:
is this needed?

Reply:
Good call - I've un-commented the parts that look useful, and removed the ones that aren't relevant